compiler/rustc_hir_typeck/src/expr.rs

```diff
@@ -2111,13 +2113,14 @@ impl<'a, 'tcx> FnCtxt<'a, 'tcx> {
         variant: &'tcx ty::VariantDef,
         access_span: Span,
     ) -> Vec<Symbol> {
+        let body_def_id = self.tcx.hir().local_def_id_to_hir_id(self.body_id);
```

Comment:
The naming is a bit off here. `body_owner_hir_id`?

Reply:
yep

Reply:
This hasn't been addressed